.github/API_REVIEW.md

```diff
@@ -24,46 +24,46 @@ possible to omit some of the sections below.
 ```md
 #### Summary
 
-> A brief of the new API, including a code sample.
-> Consider where this feature would fit into our documentation,
-> and what the updated documentation would look like.
+> A brief of the new API, including a code sample. Consider where this feature
+> would fit into our documentation, and what the updated documentation would
+> look like.
 
 <!-- TODO -->
 
 #### Motivation
 
-> Describe the problem you are trying to solve with this API change.
-> What does this API enable that was previously not possible?
+> Describe the problem you are trying to solve with this API change. What does
+> this API enable that was previously not possible?
 
 <!-- TODO -->
 
 #### Similar APIs
 
-> Is this new API similar to an existing Stripe API?
-> Are there similar APIs or prior art in other popular projects?
+> Is this new API similar to an existing Stripe API? Are there similar APIs or
+> prior art in other popular projects?
 
 <!-- TODO -->
 
 #### Alternatives
 
-> How else could we implement this feature?
-> Are there any existing workarounds that would offer the same functionality?
-> Why should we chose this implementation over another?
```

Comment:
For the record, this is another reason why I'm not a fan of prettier for Markdown. The newlines were significant before. Each question was on its own line. Compare:

> How else could we implement this feature?
> Are there any existing workarounds that would offer the same functionality?
> Why should we chose this implementation over another?

versus

> How else could we implement this feature? Are there any existing workarounds that would offer the same functionality? Why should we chose this implementation over another?

Reply:
https://prettier.io/docs/en/options.html#prose-wrap

Reply:
Per DM, we agreed that `proseWrap: always` is still the way to go, since we do want to have line wrapping in other parts of Markdown files.

Reply:
Yeah, in order of severity, these are my grievances with pretty-printing Markdown:

1. lines longer than 80 characters should be hard wrapped
1. lines full sentences on their own line should be left alone
1. two adjacent empty lines should be preserved

So prettier solves (1) at the expense of (2) and (3).